paint-timing/fcp-only/fcp-bg-image.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP due to background image</title>
```

Comment:
Does this test anything that fcp-bg-image-two-steps (which should probably be renamed) doesn't? We could probably remove.

Reply:
ok